stdlib/builtins.pyi

```diff
@@ -2006,9 +2006,9 @@ if sys.version_info >= (3, 10):
     class EncodingWarning(Warning): ...
 
 if sys.version_info >= (3, 11):
-    _BaseExceptionT_co = TypeVar("_BaseExceptionT_co", bound=BaseException, covariant=True)
+    _BaseExceptionT_co = TypeVar("_BaseExceptionT_co", bound=BaseException, covariant=True, default=Exception)
```

Comment:
This should be BaseException.

Also, this seems tricky enough that I think it's worth adding some tests.